playbooks/adhoc/openshift_hosted_logging_efk.yaml

```diff
@@ -13,4 +13,6 @@
   - import_role:
       name: openshift_logging
       tasks_from: update_master_config
-    when: openshift_hosted_logging_deploy | default(false) | bool
+    when:
+    - openshift_hosted_logging_deploy | default(false) | bool
+    - not openshift.common.version_gte_3_9
```

Comment:
We should also update the console config map when 3.9+

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_logging/tasks/install_logging.yaml#L320-L329

Reply:
I think this playbook should just be deleted actually. @ewolinetz do you know of anyone that'd be using playbooks/adhoc/openshift_hosted_logging_efk.yaml rather than playbooks/openshift-logging/config.yaml ?

Reply:
It should be deleted, all docs point to the other logging playbook for installation.